modules/install-roxctl-cli-macos.adoc

```diff
@@ -7,16 +7,20 @@
 
 You can install the `roxctl` CLI binary on macOS by using the following procedure.
 
+[NOTE]
+====
+`roxctl` CLI for macOS is available for the `amd64` architecture.
+====
+
 .Procedure
 
 . Download the `roxctl` CLI:
 +
 [source,terminal,subs=attributes+]
 ----
-$ curl -f -O https://mirror.openshift.com/pub/rhacs/assets/{rhacs-version}/bin/Darwin/roxctl
+$ curl -f -o https://mirror.openshift.com/pub/rhacs/assets/{rhacs-version}/bin/Darwin/roxctl
```

Comment:
Here it must be either `-o roxctl`
```
$ curl -f -o roxctl https://mirror.openshift.com/pub/rhacs/assets/{rhacs-version}/bin/Darwin/roxctl
```

or `-O`
```
$ curl -f -O https://mirror.openshift.com/pub/rhacs/assets/{rhacs-version}/bin/Darwin/roxctl
```

The lowercase `-o` without argument won't lead to the expected result.

I suggest to keep the line as it was.